File: src/hello_java/c_for/Ex_10871.java
package hello_java.c_for;

import java.io.*;
import java.util.StringTokenizer;

/**
 * #10871 X보다 작은 수
 *
 * 정수 N개로 이루어진 수열 A와 정수 X가 주어진다. 이때, A에서 X보다 작은 수를 모두 출력하는 프로그램을 작성하시오.
 */

public class Ex_10871 {
    public static void main(String[] args) {
        try {
            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

            StringTokenizer tokenizer = new StringTokenizer(br.readLine());

            int n = Integer.parseInt(tokenizer.nextToken());
            int x = Integer.parseInt(tokenizer.nextToken());

            tokenizer = new StringTokenizer(br.readLine());

            for (int i = 0; i < n; i++) {
                int num = Integer.parseInt(tokenizer.nextToken());
                if (num < x) bw.write(num + " ");
            }

            br.close();
            bw.flush();
            bw.close();
        } catch (IOException e) {
            System.out.print(e.getMessage());
        }
    }
}

File: src/hello_kotlin/c_for/Ex_10871.kt
package hello_kotlin.c_for

import java.io.*
import java.util.*

/**
 * #10871 X보다 작은 수
 *
 * 정수 N개로 이루어진 수열 A와 정수 X가 주어진다. 이때, A에서 X보다 작은 수를 모두 출력하는 프로그램을 작성하시오.
 */

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val writer = BufferedWriter(OutputStreamWriter(System.out))

    var tokenizer = StringTokenizer(readLine())

    val n = tokenizer.nextToken().toInt()
    val x = tokenizer.nextToken().toInt()

    tokenizer = StringTokenizer(readLine())

    for (i in 1..n) {
        val num: Int = tokenizer.nextToken().toInt()
        if (num < x) writer.write("$num ")
    }

    writer.flush()
    writer.close()
    close()
}